docs/how-to/index.md

```diff
@@ -1,11 +1,11 @@
 # How-to guides
 
-These guides will walk you through every step of using Pebble through its complete operations lifecycle.
+These guides walk you through the operations lifecycle of Pebble.
```

Comment:
Thanks -- I agree what we had wasn't great (it certainly isn't yet "complete" or "every step"). However, I don't really like the "operations lifecycle" jargon or really know what it means. What about something like:
```suggestion
These guides walk you through key operations you can perform with Pebble.
```